### PR TITLE
fixes: move title to children in CategoryLink, update URLs in footer links to social networks

### PR DESCRIPTION
**Pull request by tetianaman**
1. Component CategoryLink: prop `title` removed, `children` added.
2. Footer: social network links updated.

### CODE REVIEW

File: src/components/footer/footer.tsx [hunk 1]
```diff
@@ -92,17 +83,32 @@ export const Footer = ({ className }: FooterProps) => {
                 </ul>
                 <ul>
                     <li>
-                        <Link to={ROUTES.home.to()} className={styles.navItem}>
+                        <Link
+                            to="https://www.facebook.com/WixStudio"
+                            className={styles.navItem}
+                            target="blank"
+                            rel="noreferrer"
```
Comment (alisey) [inline src/components/footer/footer.tsx:90]: Why noreferrer?

File: src/components/footer/footer.tsx [hunk 2]
```diff
@@ -92,17 +83,32 @@ export const Footer = ({ className }: FooterProps) => {
                 </ul>
                 <ul>
                     <li>
-                        <Link to={ROUTES.home.to()} className={styles.navItem}>
+                        <Link
+                            to="https://www.facebook.com/WixStudio"
+                            className={styles.navItem}
+                            target="blank"
```
Comment (alisey) [inline src/components/footer/footer.tsx:89]: It's not `blank`, it's `_blank`.

File: src/components/footer/footer.tsx (hunk 1 above)
Comment (tetianaman) [reply src/components/footer/footer.tsx:90]: We don't  know if we can trust the site user wants to link to. I think lets keep referrer private.

Comment (tetianaman) [reply src/components/footer/footer.tsx:90]: There are a lot of resources which advise to add this attribute. All links in Codux with target="_blank" uses it.
I also cannot remove it in the project without [adding eslint rule](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md#rule-details)
Screenshot 2024-09-17 at 09 20 26

Comment (alisey) [reply src/components/footer/footer.tsx:90]: > There are a lot of resources

Those lots of resources at at least 10 years old and apply only to Internet Explorer.

> I also cannot remove it in the project without [adding eslint rule]

Not adding, removing. It's already added and it's completely useless.

> We don't know if we can trust the site user wants to link to. I think lets keep referrer private.

So this has nothing to do with the trust.

**Review by alisey — APPROVED** [review]

File: .eslintrc.cjs [hunk 3]
```diff
@@ -6,81 +6,82 @@
...
+                },
+            },
+            extends: [
+                'plugin:@typescript-eslint/recommended',
+                'plugin:import/recommended',
+                'plugin:import/typescript',
+            ],
+            rules: {
+                '@typescript-eslint/no-explicit-any': 'off',
+                'react/jsx-no-target-blank': 'off',
```
Comment (tetianaman) [inline .eslintrc.cjs:75]: this added, other just formatting

**Pull request merged**